.github/workflows/iroha2-profiling-image.yml

```diff
@@ -31,6 +18,12 @@ jobs:
       image: hyperledger/iroha2-ci:nightly-2024-04-18
     steps:
       - uses: actions/checkout@v4
+      - name: Get the release tag
+        run: |
+          RELEASE_VERSION=${{ github.event.release.tag_name }}
+          PREFIX='v'
+          TAG=${RELEASE_VERSION#$PREFIX}
```

Comment:
does this strip `v` prefix? from `RELEASE_VERSION`?

Reply:
Yes, I checked in bash